db/1.9/modules/ROOT/edits/database-connector-query-single.adoc

```diff
@@ -0,0 +1,215 @@
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('IT', 'Stephen King', 'A.Z. Fell and Co.', '9781508297123', 20);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('The Nice and Accurate Prophecies of Agnes Nutter', 'Agnes Nutter', 'A.Z. Fell and Co.', '000000000000', 200);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('Cujo', 'Stephen King', 'A.Z. Fell and Co.', '9781501192241', 20);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('Nation', 'Terry Pratchett', 'A.Z. Fell and Co.', '9780552557795', 30);
+INSERT INTO BOOK(title, author, bookshopName, isbn, price) VALUES('The Ocean at the End of the Lane', 'Neil Gaiman', 'A.Z. Fell and Co.', '9780062459367', 30);
+----
+
+== Configure a MySQL Database Connection
+
+After you create the table, go to Anypoint Studio to configure the MySQL database connection:
+
+. Create a new Mule project in Studio.
+. Navigate to the *Global Elements* view.
+. Click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field, type `http`, select *HTTP Listener config*, and click *OK*. +
+This configuration is for the HTTP *Listener* source that initiates the Mule application flow.
+. In the *HTTP Listener config* window, complete the following fields:
+
+* *Protocol* +
+`HTTP (Default)` +
+* *Host* +
+`All Interfaces [0.0.0.0] (default)` +
+* *Port* +
+`8081`
+
+[start=4]
+. Click *OK*.
+. In the *Global Elements* view, click *Create* to open the *Choose Global Type* view.
+. In the *Filter* field, type `database`, select *Database Config*, and click *OK*.
+. In the *Database Config* window, set the *Name* field to `dbConfig`.
+. In the *Connection* field, select *MySQL Connection*.
+. Click *Configure* to add the required MySQL JDBC driver and select one of: +
++
+* *Add recommended library* +
+Install the recommended library.
+* *Use local file* +
+Install the library using a local file.
+* *Add Maven dependency* +
+Install a Maven dependency to add to the project.
++
+[start=10]
+. In the *Connection* section, complete the following fields: +
++
+* *Host* +
+`localhost`
+* *Port* +
+`3306`
+* *User* +
+`root`
+* *Password* +
+`mysql`
+* *Database* +
+`sys`
++
+[start=11]
+. On the *Advanced* tab, set the *Pooling profile* field to *Edit inline*.
+. Set the *Max pool size* field to `5`.
+. Click *Test Connection* to confirm that Mule can connect to the database.
+. Click *OK*.
+
+The following screenshot shows the *HTTP Listener* global element configuration in Studio:
+
+.HTTP Listener configuration
+image::database-querysingle-example-1.png[HTTP Listener configuration]
+
+The following screenshots show the Database global element configuration in Studio:
+
+.Database Config General cofiguration
+image::database-querysingle-example-2.png[Database Config General configuration setting Host, Port, User, Password Database values parameters]
+
+== Create, Run and Test the Mule Application
```

Comment:
```suggestion
== Create, Run, and Test the Mule Application
```

Reply:
Note that this topic was listed as two in the intro: first create, then run and test.

Reply:
Oh dear...and this topic adds "configure" as well.
Please standardize the action list with the subtopics create, configure, run, and test.
Maybe each should be its own Level 2 topic?